### DISCUSSION TITLE
TweakScale found FATAL error issue

### DISCUSSION THREAD

**Discussion by TimonSE56** [TweakScale/TweakScale#302] [Support]
After I adding some mods, this error appears. I ve never seen anything like this before, All of my mods were correcly working earlier. I only reinstalled KSP. Downalding from another site  doesnt help (another repacks are also doesnt). I didnt find anything about this in inthernet. (I am so sorry,but i dont know exactly where i can find a ksp log, I found only this, hope that is ksp log)@Lisias

![Снимок экрана (8)](https://github.com/TweakScale/TweakScale/assets/133976417/37c23b37-503b-4eda-a104-138ed18af238)
![Снимок экрана (7)](https://github.com/TweakScale/TweakScale/assets/133976417/6ef20666-0ed8-4267-a431-d01cb1ec9d20)
![Снимок экрана (6)](https://github.com/TweakScale/TweakScale/assets/133976417/7a2fbeb0-ed46-4195-94da-1c5d82da2924)
![Снимок экрана (5)](https://github.com/TweakScale/TweakScale/assets/133976417/81f8fccb-db3f-4d5c-87ed-629922c9b447)

**Comment by Lisias** [comment]
Hi, @TimonSE56 !

Someone in your recent update screwed up a patch, and TweakScale detected the problem and yelled about.

Not every single bad patch is going to screw the game, but some really will and since there's no way to tell one from the another, I choose to warn you about the problem.

The KSP.log should be found in the same place where the EXE file is - and since you found some other logs, you should had found it. This is pretty weird - are you using Steam? If yes, did you got rid of the PD Launcher?

In a way or another, the last add'on I'm aware that got a bad patch is BDArmory. Try to roolback BDArmory to the previous version and see if it fixes the problem for you.

**Comment by TimonSE56** [comment]
Hi, I'm sorry I didn't answer while you were texting me, I already had a
night and I was asleep. I use steam, (but I downloaded ksp from a pirate
torrent site) I didn't get rid of any PD Launcher. I just checked, you're
right-the problem turned out to be in BD armory. I deleted it and the error
disappeared.Thank you very much!

сб, 20 мая 2023 г., 01:58 Lisias ***@***.***>:
